cupy/random/generator.py

```diff
@@ -632,6 +632,10 @@ def _interval(self, mx, size):
             n_rem -= n_succ
 
         assert n_rem == 0
+
+        if ret is None:
```

Comment:
This is nice and a clever idea,
but in order of facilitate readability and be consistent with the style this check could be done at the top of the function with:

```python
if type(size) is tuple and 0 in size
```

Reply:
Also please add an assert here `assert ret is not None` (thanks @asi1024 )

Reply:
Thanks for the review @emcastillo -- fix in 080eff0f06e90dddc3ba76c0ad553df0d3febe88